webapp-django/crashstats/crashstats/models.py

```diff
@@ -48,6 +51,8 @@ def config_from_configman():
         definition_source=[
             ElasticsearchConfig.required_config,
             PostgreSQLCrashStorage.required_config,
+            ReprocessingOneRabbitMQCrashStore.required_config,
```

Comment:
here's the magic so that it picks up the host, password etc from `SOCORRO_IMPLEMENTATIONS_CONFIG`.